src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -563,7 +567,13 @@ public AdsBaseObject verifyProvidersChain(String slotName, String providers) {
             if (expectedProviders.equals(Joiner.on("; ").join(getProvidersChain(slotName)))) {
               return getProvidersChain(slotName);
             }
-            return Collections.emptyList();
+            return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1168)